src/macaron/slsa_analyzer/provenance/loader.py

```diff
@@ -11,7 +11,7 @@
 import zlib
 from urllib.parse import urlparse
 
-from pypi_attestations import Attestation
+from cryptography import x509
```

Comment:
I think we should add this into the dependency section in `pyproject.toml`.

Reply:
Am I correct to assume that we don't need the `pypi_attestations` library anymore ? 
If that is True we might want to remove `pypi_attestations` from `pyproject.toml` too.